### ISSUE TITLE
Changelog

### ISSUE THREAD

**Issue by maxcodes** [Shopify/graphql-batch#79]
Does this project have a changelog? I want to update from 0.3.3 to 0.3.9 and I just want to make sure nothing breaks. 

Thanks!

**Comment by dylanahsmith** [comment]
No, but you can use https://github.com/Shopify/graphql-batch/compare/v0.3.3...v0.3.9 if you are interested in all the changes.

There should be no breaking changes until v0.4.0.

**Comment by stanishev** [comment]
> There should be no breaking changes until v0.4.0.

@dylanahsmith does this imply that there _are_ breaking changes if you upgrade to v0.4+ ?

**Comment by swalkinshaw** [comment]
Yes v0.4.0 has breaking changes. A few (very old) features were removed: https://github.com/Shopify/graphql-batch/releases/tag/v0.4.0

**Comment by dylanahsmith** [comment]
Just to be clear, I'm not against adding a changelog.

Also, perhaps we should bump to `1.0.0` in the future to indicate that we do use version numbers to indicate breaking changes.  Currently, we use an unstable version format `0.X.Y` where X is bumped for breaking changes and Y is bumped for any other changes.  We also try to deprecate features such that the breaking changes are just removals of a deprecated feature rather than a change that will break code depending on the library.
